sdk/python/v1beta1/kubeflow/katib/api/katib_client.py

```diff
@@ -288,6 +297,21 @@ class name in this argument.
                 https://github.com/kubernetes-client/python/blob/master/kubernetes/docs/V1EnvVar.md)
                 or a kubernetes.client.models.V1EnvFromSource (documented here:
                 https://github.com/kubernetes-client/python/blob/master/kubernetes/docs/V1EnvFromSource.md)
+            storage_per_trial: List of storage configurations for each trial container.
+                Each element in the list should be a dictionary with two keys:
+                - volume: Either a kubernetes.client.V1Volume object or a dictionary
+                  containing volume configuration with required fields:
+                  - name: Name of the volume
+                  - type: One of "pvc", "secret", "config_map", or "empty_dir"
```

Comment:
```suggestion
                  - type: One of "pvc", "secret", "configmap", or "empty_dir"
```

Usually, we'll name it as `configmap` instead of `config_map`:)